composer.json

```diff
@@ -1,23 +1,31 @@
 {
-  "name": "facebook/webdriver",
-  "description": "A php client for WebDriver",
+  "name": "markuskleint/webdriver",
```

Comment:
dont worry. I didnt want to pull-request to the original code, that was done by mistake.
I was new to building own dependencies and first thought was that the name has to match the repository.